File: branches/Medusa/Source/Fantasy.Jobs/InvalidConditionException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Fantasy.Jobs
{
 

    [Serializable]
    public class InvalidConditionException : Exception
    {
        public InvalidConditionException(string message, Exception innerException)
            : base(message, innerException)
        {

        }

        public InvalidConditionException(string message)
            : base(message)
        {

        }
    }
}

File: Fantasy.Jobs/src/fantasy/jobs/InvalidConditionException.java
package fantasy.jobs;

import java.io.Serializable;

public class InvalidConditionException extends RuntimeException implements Serializable
{
	/**
	 * 
	 */
	private static final long serialVersionUID = -3575876848171398799L;

	public InvalidConditionException(String message, RuntimeException innerException)
	{
		super(message, innerException);

	}

	public InvalidConditionException(String message)
	{
		super(message);

	}
}